src/app/main/component/events/components/events-list/events-list.component.ts

```diff
@@ -212,6 +212,14 @@ export class EventsListComponent implements OnInit, OnDestroy {
     return cities;
   }
 
+  private getStatusesForFilter(): OptionItem[] {
+    if (!!this.userId) {
```

Comment:
use ternary operator here
return !!this.userId ? eventStatusList : eventStatusList.slice(0, 2)